Dockerfile

```diff
@@ -42,6 +42,8 @@ RUN ldconfig
 COPY --from=dependencies /usr/local/plugins/ /usr/local/plugins/
 COPY --from=dependencies /usr/local/lib/ /usr/local/lib/
 COPY --from=dependencies /usr/local/bin/ /usr/local/bin/
+COPY --from=dependencies /usr/lib/ /usr/lib/
```

Comment:
Do we have some lib from this folder that is relevant to this PR?

Reply:
Some libraries installed using apt-get either install in /usr/local/ or /usr/ in my experience (not sure how this is determined but I think it's some part of the debian package setup). When we where only copying /usr/local/ our CDASimAdapter plugin would throw an 
(unknown source ) stoul exception on lines of code that used std::stoul (See related issue for this PR). After adding this the error went away. I am assuming some part of the C++ librarywas included under /usr/lib/ during the build  but then did not make it into the next run image. It still looks to me that it should have worked since we install the same library anyways using the scripts but something didn't work. I put a issue up to further fix this Docker image since right now it is a little weird (https://github.com/usdot-fhwa-OPS/V2X-Hub/issues/527)